src/diffusers/schedulers/scheduling_consistency_models.py

```diff
@@ -0,0 +1,357 @@
+# Copyright 2023 NVIDIA and The HuggingFace Team. All rights reserved.
```

Comment:
I don't think copyright is under nvidia

Reply:
Good catch, I think I copied that from another scheduler. Removed the "NVIDIA" part.